src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -365,19 +365,9 @@ private bool LaunchHost(TestProcessStartInfo testHostStartInfo, CancellationToke
             this.testHostProcessStdError = new StringBuilder(0, CoreUtilities.Constants.StandardErrorMaxLength);
             EqtTrace.Verbose("Launching default test Host Process {0} with arguments {1}", testHostStartInfo.FileName, testHostStartInfo.Arguments);
 
-            if (this.customTestHostLauncher == null)
```

Comment:
@cvpoienaru this looks like the reason why the custom test host launcher tests fails.

Reply:
Thanks Jakub. Took care of it.